subworkflows/local/scatter_genome.nf

```diff
@@ -8,16 +8,16 @@ include { GATK4_SPLITINTERVALS } from '../../modules/nf-core/gatk4/splitinterval
 workflow SCATTER_GENOME {
 
     take:
-        ch_dict   // channel: [mandatory] [ val(meta), path(dict) ]
-        ch_fai    // channel: [mandatory] [ val(meta), path(fai) ]
-        ch_fasta  // channel: [mandatory] [ val(meta), path(fasta) ]
+        ch_genome_dict   // channel: [mandatory] [ val(meta), path(dict) ]
```

Comment:
```suggestion
        ch_genome_dictionary   // channel: [mandatory] [ val(meta), path(dict) ]
```
It is what it is called in other parts of the pipeline. I think that both are nice readable but we should be consistent

Reply:
Good catch!